src/Hosting/TestHost/test/TestClientTests.cs

```diff
@@ -232,6 +232,58 @@ public async Task WebSocketWorks()
             clientSocket.Dispose();
         }
 
+[Fact]
+public async Task WebSocketSubProtocolsWorks()
```

Comment:
Formatting: indent this whole method.

Reply:
Doh! I untabbed to copy/paste into the issue and must have forgotten to re-tab. Will do! :+1: